README.md

```diff
@@ -1,9 +1,125 @@
-Freesound is a project by the Music Technology Group (MTG, http://www.mtg.upf.edu), Universitat Pompeu Fabra
+![Freesound](media/images/logo_bw.png)
 
-All the source code in this repository is licensed as GNU Affero (see _LICENSE folder).
+This repository contains the source code of the [Freesound](https://freesound.org) website.
 
-For instructions on how to set up Freesound for development, [see the wiki](https://github.com/MTG/freesound/wiki/Development-setup)
-
-For notes about development, see DEVELOPERS.md
+Freesound is a project by the [Music Technology Group](http://www.mtg.upf.edu) (MTG), [Universitat Pompeu Fabra](http://upf.edu) (UPF).
 
 [![Build Status](https://travis-ci.org/MTG/freesound.svg?branch=master)](https://travis-ci.org/MTG/freesound)
+
+
+## License
+
+All the source code in this repository is licensed as GNU Affero. Some of the dependencies might have their own licenses. See the [_LICENSE](https://github.com/MTG/freesound/tree/master/_LICENSE) folder for more details.
+
+
+## Authors
+
+For a list of authors please check out the [contributors](https://github.com/MTG/freesound/graphs/contributors) page.
+
+
+## Development
+
+Freesound is composed by a numnber of different services which can be run and orchestrated using Docker . The main service is provided by the `web` container which runs the Freesound Django application. Check out [this blog post](https://opensource.creativecommons.org/blog/entries/freesound-intro/) for some information about the Freesound technology stack. If you're to do development for Freesound, please checkout the [DEVELOPERS](https://github.com/MTG/freesound/blob/master/DEVELOPERS.md) file for some guidelines. 
+
+Below are instructions for setting up a local Freesound installation for development. It is assumed that you have a working Docker installation.
+
+
+### Setting up Freesound for development in 13 easy steps
+
+1. Clone source code repository
+
+  ```
+git clone git@github.com:MTG/freesound.git
+cd freesound
+```
+
+2. Create a directorty named `freesound-data` inside the repository folder
+
+  ```
+mkdir freesound-data
+```
+
+3. Download the Freesound development data zip file (~20GB) and uncompress it inside `freesound-data`. You should request this file to Freesound admins. File structure should look like this:
```

Comment:
where is the link to this file and the similarity index files?

Reply:
I added links to the files now, although we'll need to give individual permissions. This will be good to keep track of people doing development (if any!).